cypress/e2e/0-api/configuration_api.cy.js

```diff
@@ -0,0 +1,377 @@
+describe('configuration API', () => {
+  const PORT_REGEX = /[0-9]*/;
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  context('GET /config/network', () => {
+    it('retrieve network settings', () => {
+      cy.reqNetwork().then(response => {
+        const body = response.body;
+        expect(body.type).to.be.a('string');
+        if (body.type !== 'DHCP') {
+          expect(body).to.have.property('address');
+          if (body.address !== null) { expect(body.address).to.be.a('string'); }
+          expect(body).to.have.property('subnetMask');
+          if (body.subnetMask !== null) { expect(body.subnetMask).to.be.a('string'); }
+          expect(body).to.have.property('gateway');
+          if (body.gateway !== null) { expect(body.gateway).to.be.a('string'); }
+          expect(body).to.have.property('dns');
+          if (body.dns !== null) { expect(body.dns).to.be.a('string'); }
+        }
+      });
+    });
+  });
+
+  context('GET /config/hostname', () => {
+    it('retrieve hostname', () => {
+      cy.reqHostname().then(response => {
+        const body = response.body;
+        expect(body.hostname).to.be.a('string');
+      });
+    });
+  });
+
+  context('GET /config/ntp', () => {
+    it('retrieves NTP client settings', () => {
+      cy.reqNTPClient().then(response => {
+        const body = response.body;
+        body.ntpServers.forEach(serverName => {
+          expect(serverName).to.be.a('string');
+        });
+        expect(body.timezone).to.be.a('string');
```

Comment:
The object destructuring becomes really handy when you have many properties and you don't want to keep writing `body.whatever`...
```suggestion
        const {
          ntpServers,
          timezone,
        } = response.body;
        ntpServers.forEach(serverName => {
          expect(serverName).to.be.a('string');
        });
        expect(timezone).to.be.a('string');
```
While it is the same number of lines there's less repetition and if you added a new property later in `body` it's very easy to add a new line in the destructuring expression.